### ISSUE TITLE
Request access to and Download multiple datasets at the same time

### ISSUE THREAD

**Issue by stevenmce** [IQSS/dataverse#4480]
As a user, I would like to be able to request access to content from several restricted access datasets at the same time (e.g. all datasets within a Dataverse such as our "Australian Survey of Social Attitudes" - [https://dataverse.ada.edu.au/dataverse/aussa](https://dataverse.ada.edu.au/dataverse/aussa)). Currently users have to request each dataset - or more accurately, files within each dataset - one by one.

I expect that this will also be relevant to issue #4246

**Comment by pdurbin** [comment]
@stevenmce yes, #4246 is where the "shopping cart" for the "Compute" button was introduced. It's a similar concept.

**Comment by cmbz** [comment]
2024/09/16: Updated title, keeping to investigate.

**Comment by pdurbin** [comment]
"The [#4480](https://github.com/IQSS/dataverse/issues/4480) issue from Steve is very similar to our wish." -- @LauraHuisintveld in [Zulip](https://dataverse.zulipchat.com/#narrow/channel/375707-community/topic/Interest.20in.20developing.20a.20.27shopping.20basket.27.20for.20datasets.3F/near/609438048).
